Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM docker.io/library/rockylinux:9@sha256:d7be1c094cc5845ee815d4632fe377514ee6ebcf8efaed6892889657e5ddaaa6 as rockylinux9
+FROM registry.access.redhat.com/ubi9/ubi@sha256:763f30167f92ec2af02bf7f09e75529de66e98f05373b88bef3c631cdcc39ad8 as ubi
```

Comment:
**commit message:**
Can we improve the commit message so that it explicitly states that the only reason we're switching to it is to enable access to entitled content via subscription manager that would not otherwise be possible via the current Rocky Linux image?